js/templates/modals/settings/moderation.html

```diff
@@ -164,4 +164,10 @@ <h2 class="h3 clrT">Moderation</h2>
       </div>
     </div>
   </div>
+
+  <div class="contentBox padMd clrP clrBr clrSh3">
```

Comment:
If the other save button uses the processing template, this one should too to stay consistent. 

You should be able to just copy and paste the other one in the template.